db/migrate/20230502102124_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
```

Comment:
- [ ] Let's make sure that the data in our migration files reflect what we have on the ERD. In this case, we want to name our user foreign keys as __author_id__. By default, this can bring issues because Rails only knows a user (user_id). We need to explicitly tell Rails that a user will be called __author__ and its __id__ will now be __author_id__, __HINT__:
```ruby
    t.references :author, foreign_key: { to_table: 'users' } # This will create author_id and a corresponding index 
```
[Read this documentation for reference](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)

Reply:
@elisha2kyakpo1 ERD diagram says column name is `user_id` and not `author_id` for table `comments`. I have added users(id) as foreign key. Don't understand what you are suggesting here

![image](https://user-images.githubusercontent.com/116718155/235700414-e7b2070a-150d-4a02-b132-effa3511de3d.png)

![image](https://user-images.githubusercontent.com/116718155/235700714-a9e6df5b-a9a0-4b15-823a-f1c89d3a17f3.png)